diff --git a/node_modules/dcmjs/build/dcmjs.es.js b/node_modules/dcmjs/build/dcmjs.es.js
index 690a8be..ce10005 100644
--- a/node_modules/dcmjs/build/dcmjs.es.js
+++ b/node_modules/dcmjs/build/dcmjs.es.js
@@ -12543,7 +12543,7 @@ function checkSEGsOverlapping(pixelData, multiframe, imageIds, validOrientations
     var imageId = findReferenceSourceImageId(multiframe, frameSegment, imageIds, metadataProvider, tolerance);
 
     if (!imageId) {
-      console.warn("Image not present in stack, can't import frame : " + i + ".");
+      console.warn("Image not present in stack, can't import frame : " + frameSegment + ".");
       return "continue";
     }
 
